content/6-form-input/3-radio/marko/PickPill.marko

```diff
@@ -0,0 +1,22 @@
+function handleChange (event) {
+  picked = event.target.value;
+}
+
+<let/picked = "red"/>
```

Comment:
```suggestion
<let/picked = "red"/>
<const/handleChange(event) {
  picked = event.target.value;
}/>
```

`function` at this point in the template would be parsed as a custom tag, and it can't be `static function` because we're assigning to the instance variable.

Reply:
I was _wondering_ why the syntax highlighter wasn’t upset